fv3core/stencils/updatedzd.py

```diff
@@ -77,10 +78,19 @@ def zh_damp_stencil(
     fy2: FloatField,
     rarea: FloatFieldIJ,
     zh: FloatField,
+    zs: FloatFieldIJ,
+    ws: FloatFieldIJ,
+    dt: float,
 ):
     with computation(PARALLEL), interval(...):
         zhbase = zh_base(z2, area, fx, fy, ra_x, ra_y)
         zh[0, 0, 0] = zhbase + (fx2 - fx2[1, 0, 0] + fy2 - fy2[0, 1, 0]) * rarea
+    with computation(BACKWARD):
+        with interval(-1, None):
+            ws[0, 0, 0] = (zs - zh) * 1.0 / dt
+        with interval(0, -1):
+            other = zh[0, 0, 1] + DZ_MIN
+            zh[0, 0, 0] = zh if zh > other else other
```

Comment:
why are we putting 0's on the lhs?

Reply:
like above, we don't have consistency and this code was just copied